keyserver/handler.go

```diff
@@ -19,10 +19,22 @@ func (server *ConiksServer) handleRegistrationMessage(reg *p.RegistrationRequest
 			p.Error(p.ErrorMalformedClientMessage)
 	}
 
+	server.Lock()
+	// check the temporary bindings array first
+	// currently the server allows only one registration/key change per epoch
+	if server.tbs[reg.Username] != nil {
```

Comment:
Not sure, if this is a good idea: But wouldn't it make more sense to return the already existing TB instead of an error here?